frontend/src/upload.js

```diff
@@ -253,32 +293,41 @@ $(document).ready(function() {
         toggleHeader();
       });
     });
+    document.getElementById('delete-file').onclick = () => {
```

Comment:
When I click "delete file", it takes me to the home page that lists all the files. at first, my deleted file will be there, then I can see it visually update to remove it. It should be gone from the beginning.